cymetric/root_metrics.py

```diff
@@ -34,6 +39,7 @@ def __call__(self, conds=None, *args, **kwargs):
 
 
 def root_metric(obj=None, name=None, schema=None, *args, **kwargs):
+    """Checks for existence of root metric."""
```

Comment:
I see that in America's dairyland cheese is more important than knowledge :)

On Tue, Mar 17, 2015 at 2:49 PM opotowsky <notifications@github.com> wrote:

> In cymetric/root_metrics.py
> <https://github.com/cyclus/cymetric/pull/47#discussion_r26609079>:
>
> > @@ -34,6 +39,7 @@ def __call__(self, conds=None, *args, **kwargs):
> >
> >
> >  def root_metric(obj=None, name=None, schema=None, *args, **kwargs):
> > +    """Checks for existence of root metric."""
>
> I need to stop buy fancy cheese with all my textbook allowance! ;-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/cymetric/pull/47/files#r26609079>.
>